src/status_im2/contexts/onboarding/notifications/views.cljs

```diff
@@ -0,0 +1,71 @@
+(ns status-im2.contexts.onboarding.notifications.views
+  (:require
+    [quo2.core :as quo]
+    [quo2.foundations.colors :as colors]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]
+    [react-native.core :as rn]
+    [react-native.platform :as platform]
+    [status-im.notifications.core :as notifications]
+    [status-im2.contexts.onboarding.notifications.style :as style]
+    [status-im2.contexts.onboarding.common.background :as background]))
+
+(defn page-title
+  []
+  [rn/view {:style style/title-container}
+   [quo/text
+    (merge {:style {:color colors/white}}
+           style/title-text-style)
+    (i18n/label :t/intro-wizard-title6)]
+   [quo/text
+    (merge {:style {:color colors/white}}
+           style/subtitle-text-style)
+    (i18n/label :t/enable-notifications-sub-title)]])
+
+(defn navigate-back
+  [root]
+  [rn/view {:margin-top (if platform/ios? 44 0)}
```

Comment:
again this 44, why do we add it everywhere?

Reply:
@flexsurfer I have replaced this with `status-bar-height`

Reply:
@flexsurfer  every page that relies on the page-nav needs this `:padding-top 44` because of the status-bar-height on ios